src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolumeHelperCommon.h

```diff
@@ -344,6 +344,7 @@ static int vtkTrilinearInterpolation(F *point,
           break;
         }
         inPtrTmp++;
+        importancePtr++;

```

Comment:
You are right. It does not need to be modified at all, as this whole function processes just one input pixel (and each importance mask pixel corresponds to one input pixel).